taichi/transforms/lower_ast.cpp

```diff
@@ -82,6 +82,21 @@ class LowerAST : public IRVisitor {
     }
   }
 
+  void visit(FrontendFuncCallStmt *stmt) override {
+    auto block = stmt->parent;
+    auto ident = stmt->ident;
+    TI_ASSERT(block->local_var_to_stmt.find(ident) ==
+              block->local_var_to_stmt.end());
+    std::vector<Stmt *> args;
```

Comment:
```suggestion
    std::vector<Stmt *> args(stmt->args.exprs.size());
```

and switch the push back to indexing